src/d/actor/d_a_pw.cpp

```diff
@@ -107,8 +111,253 @@ void first_mode_change(pw_class*) {
 }
 
 /* 00002A90-00003B08       .text action_dousa__FP8pw_class */
-void action_dousa(pw_class*) {
+void action_dousa(pw_class* i_this) {
     /* Nonmatching */
+    daPy_py_c* link = daPy_getPlayerActorClass();
+    float randomValue;
+    switch (i_this->m368) {
+    case 0:
+        anm_init(i_this, 0x23, 7.0, '\x02', 1.0, -1);
```

Comment:
```suggestion
        anm_init(i_this, 0x23, 7.0f, J3DFrameCtrl::LOOP_REPEAT_e, 1.0f, -1);
```
all the anm_init calls should probably look something like this. the two floating-pointer numbers are singles not doubles, and the unsigned char argument is the J3DFrameCtrl::Attribute_e enum (the reason I know this is because the anm_init function is very similar across most enemies)